bertopic/_bertopic.py

```diff
@@ -152,6 +153,7 @@ def __init__(self,
         self.nr_topics = nr_topics
         self.min_topic_size = min_topic_size
         self.calculate_probabilities = calculate_probabilities
```

Comment:
Since we overwrite this parameter, it would be better to remove it from BERTopic constructor.